src/main/java/com/artipie/rpm/pkg/HeaderTags.java

```diff
@@ -260,6 +260,34 @@ public String sourceRmp() {
             .map(Flags::find).collect(Collectors.toList());
     }
 
+    /**
+     * Get the conflicts name header.
+     * @return Value of header tag CONFLICTNAME.
+     */
+    public List<String> conflicts() {
```

Comment:
Can conflicts list include duplicates? And how it will be used later? If it doesn't have duplicates and client code will check if some value contains in this list, I'd suggest changing it from list to set, because `contains` operations on set is much faster than on list.

Same could be applied to other methods.

Reply:
Duplicates will be checked later, while adding conflict list to primary (though I'm not sure duplicates are even possible here). To check the duplicate, we should consider all three conflict item properties: name, version and flag.